### PR TITLE
Robust download properties added to reference guide 

### PR DESCRIPTION
**Pull request by velitchko-valkov**
[#96] Add robust download properties to software update reference guide

- Added descriptions of the properties
- Updated the template

Signed-off-by: Velitchko Valkov <Velitchko.Valkov@bosch.io>

### CODE REVIEW

File: web/site/content/docs/references/software-update-config.md [hunk 1]
```diff
@@ -17,6 +17,9 @@ To control all aspects of the software update behavior.
 | artifactType | string | archive | Type of the artifact that is to be processed: archive or plain |
 | install | string[] | | Absolute path to the install script/command and an optional sequence of additional flags/parameters |
 | storageLocation | string | ./ | Path to the storage directory where the working files are stored |
+| **Download** | | | |
+| downloadRetryCount | int| 0 | Number of retries, in case of a failed download |
+| downloadRetryInterval | durationTime | 5s | Interval between retries, in case of a failed download |
```
Comment (e-grigorov) [inline web/site/content/docs/references/software-update-config.md:22]: dorationTime - why is it? This type is not used in the reference guide at all. All other documents just use string as a type and the description contains the special format. Could you please reuse already available descriptions?

**Review by e-grigorov — CHANGES_REQUESTED** [review]

File: web/site/content/docs/references/software-update-config.md (hunk 1 above)
Comment (velitchko-valkov) [reply web/site/content/docs/references/software-update-config.md]: Okay, I will fix the description and type.

File: web/site/content/docs/references/software-update-config.md [hunk 2]
```diff
@@ -17,6 +17,9 @@ To control all aspects of the software update behavior.
 | artifactType | string | archive | Type of the artifact that is to be processed: archive or plain |
 | install | string[] | | Absolute path to the install script/command and an optional sequence of additional flags/parameters |
 | storageLocation | string | ./ | Path to the storage directory where the working files are stored |
+| **Download** | | | |
+| downloadRetryCount | int| 0 | Number of retries, in case of a failed download |
+| downloadRetryInterval | string | 5s | Interval between retries, in case of a failed download.  Should be a sequence of decimal numbers, each with optional fraction and a unit suffix, such as '300ms', '1.5h', '10m30s', etc. Valid time units are 'ns', 'us' (or 'µs'), 'ms', 's', 'm', 'h'." |
```
Comment (e-grigorov) [inline web/site/content/docs/references/software-update-config.md:22]: All descriptions are kept short i.e. in one sentence without full stop in the end. Could you please align the format?

**Review by e-grigorov — CHANGES_REQUESTED** [review]

File: web/site/content/docs/references/software-update-config.md (hunk 2 above)
Comment (velitchko-valkov) [reply web/site/content/docs/references/software-update-config.md]: Okay, I will align it.

**Review by e-grigorov — APPROVED** [review]

**Pull request merged**